### PR TITLE
CtrlX Iteration

### PR DESCRIPTION
**Pull request by Yndira-FlowForge**
## Description

Redesigned layout and updated content for ctrlX page

## Related Issue(s)

#768

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/flowforge/flowforge/blob/main/CONTRIBUTING.md)
 - [x] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

**Comment by Yndira-FlowForge** [comment]
@iskerrett, development is "done". We will hold off on it until we have the link for the main button.
Additionally, you mentioned that you wanted to make it visible. Do you mean adding a new menu item to the nav bar?

**Comment by iskerrett** [comment]
Good point we need to decide where to make this page visible. We have a partners page that is currently for the referral partner program. We should probably look at expanding this page to include other partnerships. I will take a look and develop some suggestions.

https://flowforge.com/partners/

**Comment by ZJvandeWeg** [comment]
@MarianRaphael Do we have a due date from Rexroth?

**Comment by MarianRaphael** [comment]
I have spoken to Rexroth, and the process has started on their end. We can do nothing more but wait until their internal process is finished. I'll keep you updated.

Additionally, I presented our Rexroth partner with a preview of the website. He suggested that it would be great if we could replace the ctrlX Automation image with the ctrlX OS image due to an upcoming change in their communication strategy.

@Yndira-FlowForge , could you please update the image?
DC-AE_ctrlX_OS_Signee_12x12_dbr_Weiß_202303_RGB
DC-AE_ctrlX_OS_Signee_18x18_dbr_Weiß_202303_RGB

File: src/partners/ctrlx.njk [hunk 1]
```diff
@@ -1,221 +1,131 @@
...
-                <label class="sub-label">Included<label></div>
-                    <p class="mt-4">Need guidance on Node-RED? Seek support for your issues in the Community Forum, offering a plethora of
-                        resolutions.</p>
-                    <a class="ff-btn ff-btn--secondary uppercase" href="https://discourse.nodered.org/">Community Forum</a>
+<div class="nohero w-full ff-bg-light">
+    <!--Hero Content-->
+    <div class="bg-white w-full px-6 pt-8 md:pt-16 pb-16 md:px-0">
+        <div class="container md:m-auto text-center max-w-4xl flex flex-col items-center md:items-start">
+            <h1 style="line-height: 3.25rem; font-size: 2.5rem;" class="mt-0 md:pl-0 md:pr-3 md:text-left md:max-w-xl font-semibold leading-normal -mx-1 md:mx-0">
+                Production Ready <span class="inline-block">Node-RED</span> for ctrlX Automation
```
Comment (MarianRaphael) [inline src/partners/ctrlx.njk:16]:
```suggestion
                Production Ready <span class="inline-block">Node-RED</span> for ctrlX
```

**Review by iskerrett — APPROVED** [review]

**Comment by Yndira-FlowForge** [comment]
Hi, do we have an URL for the button on this section?

Screenshot 2023-09-05 at 12 40 56

**Comment by iskerrett** [comment]
Here is the url https://developer.community.boschrexroth.com/t5/Store-and-How-to/FlowFuse-Node-RED/ba-p/72303

**Comment by Yndira-FlowForge** [comment]
> Here is the url https://developer.community.boschrexroth.com/t5/Store-and-How-to/FlowFuse-Node-RED/ba-p/72303

@iskerrett that url redirects to https://developer.community.boschrexroth.com/t5/ctrlX-AUTOMATION-Community/ct-p/dcdev_community

In there I found this one https://developer.community.boschrexroth.com/t5/Store-and-How-to/FlowFuse-Node-RED/ba-p/82135 I think they changed it somehow and this is the new one. I'm using this one.

**Review by MarianRaphael — APPROVED** [review]

**Pull request merged**